### DISCUSSION TITLE
Gns3 multiple labs simultaneously

### DISCUSSION THREAD

**Discussion by priyanshuz** [GNS3/gns3-server#2213] [General]
Does gns3 support running multiple labs simultaneously? 
If not, then is this something possible to do in future.

**Comment by spikefishjohn** [comment]
yes it does. Just make a new project and go.

Side note, I would enable this. Super annoying otherwise (at least for bare metal gns3 installs)
File ->Edit Project -> "Leave this project running in the background when closing GNS3"
